tests/Oryx.Integration.Tests/Node/NodeDynamicInstallationTest.cs

```diff
@@ -0,0 +1,130 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.Integration.Tests
+{
+    [Trait("category", "node")]
+    public class NodeDynamicInstallationTest : NodeEndToEndTestsBase
+    {
+        private readonly string DefaultSdksRootDir = "/tmp/oryx/nodejs";
+
+        public NodeDynamicInstallationTest(ITestOutputHelper output, TestTempDirTestFixture fixture)
+            : base(output, fixture)
+        {
+        }
+
+        [Fact]
+        public async Task CanBuildAndRunApp_AfterSettingUpEnvironmentExplicitly()
```

Comment:
I would change this test from fact to theory, may be use 12 and 10 for minimal test optimization?